### ISSUE TITLE
Unanswered questions

### ISSUE THREAD

**Issue by teshenglin** [Jerry263104/2025_machine_learning#1]
Week 1: 我提的這方法是專門針對 learning rate 的調整方法, 目前只能證明他在 convex 問題上能加速 gradient descent iteration. 我不確定在一般深度學習的問題上是否也能見效. 

Week 2: 我覺得你是對的, 做未來時間的預測效果應該不會太好. 

Week 3: 目前對 activation function 的選取並沒有比較好的說法. 其實也有人用 sin 當 activation function. 不過目前應用端最常用的其實是 ReLU.
